examples/lib/defaultHtmlStepUi.js

```diff
@@ -276,6 +276,21 @@ function DefaultHtmlStepUi(_sequencer, options) {
         });
     });
 
+    $step('.color-picker-target').each(function(i, input) {
```

Comment:
I am extremely sorry for asking you to change this to `$step`. It is supposed to be `$stepAll` otherwise, it won't work for multiple pickers.
Please change it again.
I am extremely sorry. I forgot what `$stepAll` does, for a moment.

Reply:
I thought so too that's why I used stepAll but when I changed it to step still it was working fine that's why I pushed commit.

![Screenshot from 2020-01-13 00-16-55](https://user-images.githubusercontent.com/42492389/72223922-31811400-359a-11ea-941c-4a61a0ad0cda.png)

Reply:
Sorry. Please revert it though.

Reply:
Ok.